compiler_opt/rl/inlining/inlining_runner.py

```diff
@@ -71,6 +71,8 @@ def compile_fn(
       cancelled work.
       RuntimeError: if llvm-size produces unexpected output.
     """
+    if cancellation_manager is None:
```

Comment:
weird... hmm, should we just not pass `cancellation_manager` because there's `self._cancellation_manager` anyway?